lib/Util/MaxUploadSize.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\RepositoryForms\Util;
+
+class MaxUploadSize
+{
+    const BYTES = 'B';
+    const KILOBYTES = 'K';
+    const MEGABYTES = 'M';
+    const GIGABYTES = 'G';
+
+    /** @var int */
+    protected $value;
+
+    /**
+     * Return value of upload_max_filesize in bytes.
+     *
+     * @param string|null $unit
+     *
+     * @return int
+     */
+    public function get(?string $unit = null): int
+    {
+        if (null === $this->value) {
+            $this->value = $this->str2bytes(ini_get('upload_max_filesize'));
+        }
+        $value = $this->value;
+
+        switch ($unit) {
+            case self::GIGABYTES:
+                $value *= 1 / 1024;
+            case self::MEGABYTES:
+                $value *= 1 / 1024;
+            case self::KILOBYTES:
+                $value *= 1 / 1024;
+            case self::BYTES:
+            default:
+        }
+
+        return $value;
+    }
+
+    protected function str2bytes(string $str): int
```

Comment:
Naming is kinda weird on this one. We are using camelCase: `stringToBytes` feels more fitting to our conventions.